Addition to using-on-macos.md

ricardodsanchez opened the pull request:
# Add steps to add vs code to the PATH so it can be launched from the terminal

## Summary

Added details on how to open Visual Studio Code from the terminal by typing `code .`

ricardodsanchez committed:
Addition to using-on-macos.md

Many people ask how to launch Visual Studio Code from the terminal, adding this information wold be beneficial.

jzeferino commented on docs/core/tutorials/using-on-macos.md:
```diff
@@ -220,4 +230,4 @@ Open the debugger tab by selecting the Debug icon in the Visual Studio Code tool
 <a name="dotnet-restore-note"></a>
-[!INCLUDE[DotNet Restore Note](~/includes/dotnet-restore-note.md)]
\ No newline at end of file
+[!INCLUDE[DotNet Restore Note](~/includes/dotnet-restore-note.md)]
```
@ricardodsanchez why was this line changed?

ricardodsanchez replied:
It wasn't intentional or manually changed. Not sure how/why this was updated.

jzeferino commented on docs/core/tutorials/using-on-macos.md:
```diff
@@ -32,6 +32,16 @@ In this tutorial, you create three projects: a library project, tests for that l
 
 Start Visual Studio Code. Press <kbd>Ctrl</kbd>+<kbd>\`</kbd> (the backquote or backtick character) or select **View > Integrated Terminal** from the menu to open an embedded terminal in Visual Studio Code. You can still open an external shell with the Explorer **Open in Command Prompt** command (**Open in Terminal** on Mac or Linux) if you prefer to work outside of Visual Studio Code.
 
+### Launching Visual Studio Code from the command line
```
You could add a numbered list with this steps:

1.  step x
2. step y

and so on.

ricardodsanchez replied:
Great suggestion, I updated the PR.

ricardodsanchez committed:
Updated PR based on code review comments

Numbered the steps and updated the keys combination needed to open the command list in Visual Studio Code.

ricardodsanchez commented:
My latest commit shows `OpenPublishing.Build - Validation status: errors` but it doesn't tell me what the errors are. Any thoughts?

jzeferino reviewed (approved):
LGTM.

mairaw commented:
@ricardodsanchez I'll close and reopen to start a new build. That usually happens when an exception occurs or internal error.

ricardodsanchez commented:
@jzeferino Can this be approved? Or are there any other steps needed in order to approve it? Thanks.

mairaw commented:
One of us from the .NET documentation team will review your changes shortly.

jzeferino commented:
@ricardodsanchez, @mairaw answered :)

mairaw reviewed (changes requested):
Hi @ricardodsanchez, I finally got around to review this. I really don't think that the topic needs that information. embedded on it. I think it belongs on the VS Code docs, which I saw that it already does (https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line).

What could be a good addition, in my opinion, is to add a link to [Running VS Code on Mac](https://code.visualstudio.com/docs/setup/mac) into a See Also section in the end of the article like we have in other topics (e.g. https://docs.microsoft.com/en-us/dotnet/core/tutorials/using-with-xplat-cli#see-also).

mairaw commented:
@ricardodsanchez please let me know if you'd like to make the changes I'm proposing. Thanks!

ricardodsanchez commented:
@mairaw Thank you for the reply and the reminder about it. What you propose is a good idea in order to keep relevant documentation linked to each other. However, as a developer, I prefer to get the information (examples, documentation, etc) I need right when I need it, without jumping to another resource. This document `Getting started with .NET Core on macOS` is targeted to people wanting to use .NET Core on a Mac, and my suggestion is about giving them the option to launch Visual Studio Code from the terminal, which is something a developer using a Mac will want to do often. My two cents. What are your thoughts about that?

mairaw commented:
@ricardodsanchez I still think that this piece of information is a general information on how to work with VS Code. Even though it might be common for developers to do that, I think it distracts from the goal of the article. And we try to avoid to duplicate content. 

The other thing about this particular topic that always bugged me a little is that the tutorial is not a VS Code one or particular to macOS, it's mostly a CLI tutorial with an entry about debugging in the end, which we show how to do already on this topic: https://docs.microsoft.com/en-us/dotnet/core/tutorials/with-visual-studio-code#debug. 

Adding my team here to see what they think @JRAlexander @BillWagner @rpetrusha.

JRAlexander commented:
I think @mairaw is correct. While it is helpful to know that for VS Code, I'd agree that it distracts from the goal of the article.

rpetrusha commented:
I agree with @mairaw. This is a getting started tutorial, which is aimed at an entry-level developer. Information on starting Visual Studio Code from the command line is well outside the scope of the topic. It's useful information, but it properly belongs in a Visual Studio Code topic, not in a topic on getting started with .NET Core using Visual Studio Code. And of course, if we add it here, information on starting Visual Studio Code from the command line has to be changed in two places if the information changes -- and duplicated information is almost very difficult to correct.

mairaw commented:
@ricardodsanchez please let us know if you'd like to link to that VS Code article instead. Otherwise, I'll be closing this PR. Thanks so much!

mairaw commented:
Closing for now. Feel free to reopen if you'd like to address our feedback. Thank you again @ricardodsanchez for your interest in contributing to our documentation.
